.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -136,19 +137,21 @@ body:
         - "No"
     validations:
       required: true
-
-  - type: dropdown
+      
+  - type: textarea
     id: adblock
     attributes:
       label: Is this bug caused by the adblocker?
-      description: You can test this by temporarily allowing ads for the site.
-      multiple: false
-      options:
-        - "Yes"
-        - "No"
+      description: |
+        You can test this by temporarily allowing ads for the site.
+        1. No.
+        2. Yes, I have attached the AdBlock  URL and Logs generated from `chrome://crashes`.
```

Comment:
Your issue was related to the adblocker but not caused by it; this question does not need these changes as it is only about bugs caused by the adblocker.

Reply:
@uazo had recommended to ask about the AdBlock filters URL for `.dat` file when the issue is with AdBlock, that's the reason I changed that.

Reply:
> had recommended to ask about the AdBlock filter URL

I would propose a separate mandatory field only if the issue is related to the adblock

Reply:
Sure then, I'll mark this as resolved.
Thank you.